ansible/roles/debops.phpipam/tasks/phpipam.yml

```diff
@@ -4,15 +4,15 @@
 
 - name: Create phpIPAM group
   group:
-    name: '{{ phpipam_group }}'
+    name: '{{ phpipam__group }}'
     system: True
     state: 'present'
 
 - name: Create phpIPAM user
   user:
-    name: '{{ phpipam_user }}'
-    group: '{{ phpipam_group }}'
-    home: '{{ phpipam_home }}'
+    name: '{{ phpipam__user }}'
+    group: '{{ phpipam__group }}'
+    home: '{{ phpipam__home }}'
     shell: '/bin/false'
```

Comment:
The shell should be changed to `/usr/sbin/nologin` since `/bin/false` is not a valid shell by default.

Reply:
done